tensorrt_llm/llmapi/disagg_utils.py

```diff
@@ -15,6 +16,11 @@
 ]
 
 
+class ServerRole(Enum):
+    CONTEXT = 0
```

Comment:
As previously discussed, the restriction on the role needs to be lifted going forward.